policyengine_us/variables/gov/states/nj/njdhs/tanf/income/nj_tanf_countable_earned_income.py

```diff
@@ -19,16 +19,41 @@ def formula(spm_unit, period, parameters):
         months_enrolled_in_tanf = person("months_enrolled_in_tanf", period)
         weekly_hours_worked = person("weekly_hours_worked", period)
         # New Jerset Admin Code 10:90-3.8(b)
-        return where(
-            months_enrolled_in_tanf > p.first_month_threshold,
-            where(
-                months_enrolled_in_tanf > p.consecutive_month_threshold,
-                gross_earned_income * (1 - p.additional_percent),
-                where(
-                    weekly_hours_worked >= p.work_hours_threshold,
-                    gross_earned_income * (1 - p.consecutive_month_percent),
-                    gross_earned_income * (1 - p.additional_percent),
-                ),
-            ),
-            gross_earned_income * (1 - p.first_month_percent),
+        person_meet_higher_work_hours_threshold = (
+            weekly_hours_worked >= p.work_hours_threshold
         )
+        person_enrolled_in_tanf_for_first_month = (
+            months_enrolled_in_tanf <= p.first_month_threshold
+        )
+        person_enrolled_in_tanf_for_consecutive_months = (
+            months_enrolled_in_tanf <= p.consecutive_month_threshold
+        ) and (months_enrolled_in_tanf > p.first_month_threshold)
+        person_enrolled_in_tanf_for_additional_months_with_work_hours_over_20 = (
+            months_enrolled_in_tanf > p.consecutive_month_threshold
+            and person_meet_higher_work_hours_threshold
+        )
+        person_enrolled_in_tanf_for_additional_months_with_Work_hours_below_20 = (
+            months_enrolled_in_tanf > p.first_month_threshold
+        )
+        if person_meet_higher_work_hours_threshold:
```

Comment:
Since `if` statements aren't vectorized, we have to use `where` and `select` statements instead

Reply:
OK. Is there an example shows how to use "select" statement or how to use "where" and "select" to achieve the function of "if" there? Originally, we are using a nested where function, it is not so readable although it passes all the test. That's why we are trying to use "if" function.

Reply:
@xudonglai0426 see my comment in the review below